src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaHumanExperimentService.java

```diff
@@ -18,11 +20,16 @@ public HcaHumanExperimentService(ExperimentTrader experimentTrader,
         this.hcaHumanExperimentDao = hcaHumanExperimentDao;
     }
 
-    public ImmutableSet<Experiment> getPublicHumanExperiments(String characteristicName, String characteristicValue) {
-        return hcaHumanExperimentDao.fetchExperimentAccessions(characteristicName, characteristicValue)
+    public ImmutableSet<Experiment> getPublicHumanExperiments(String characteristicName,
+                                                              Set<String> characteristicValues) {
+
+        Set<String> experimentAccessions = hcaHumanExperimentDao.fetchExperimentAccessions(characteristicName,
```

Comment:
`experimentAccessions` is a throw away variable here. You can directly do 
`return hcaHumanExperimentDao.fetchExperimentAccessions(characteristicName, characteristicValues)
.stream()
....`

Reply:
Removed throw away variable.